test/emptytheme/functions.php

```diff
@@ -20,12 +20,13 @@ function emptytheme_support() {
 	add_action( 'after_setup_theme', 'emptytheme_support' );
 endif;
 
-/**
```

Comment:
I'm curious what triggered this change ? As far as I can see, this is not coming from WPCS 3.0, so I wonder if it belongs in this PR.

Reply:
You are correct, technically this does not belong in this PR since from what I can see, It's also an issue on `trunk`.
However, in this PR I fixed all CS issues that were reported. I'll change the title of the PR to reflect that 👍